masterlist.yaml

```diff
@@ -3741,9 +3737,9 @@ plugins:
       - 'Obsidian_TS_Patch_Luminous_WaterFix.esp'
       - 'Obsidian_TS_Patch_Spectral_WaterFix.esp '
       - 'Obsidian Weathers - Wander Patch.esp'
-    clean:
-      - crc: 0x795E49B4
-        util: 'SSEEdit v4.0.3'
+    msg:
+      - <<: *useInstead
+        subs: [ '[Obsidian Weathers - TrueStorms Merged Compatibility](https://www.nexusmods.com/skyrimspecialedition/mods/17198/)' ]
```

Comment:
I can't really agree with this, because either set of patches should work.
Has there been some major update for any of these mods lately?